plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -632,4 +634,60 @@
         </xsl:analyze-string>
     </xsl:template>
 
+    <xsl:template name="place-fig-lbl">
```

Comment:
@Yuliia-Bondarenko-1806 Could you please exclude whole 'fig--title-label' span instead of number only?

Reply:
done